lib/github-pages.rb

```diff
@@ -1,5 +1,5 @@
 class GitHubPages
-  VERSION = 38
+  VERSION = 39
```

Comment:
You can close this pull request, create a new branch that only has the changes you desire and then do another pull request. I did that for a few requests that had unintended changes.